src/components/services-view/services-view.html

```diff
@@ -5,14 +5,34 @@
   <div ng-if="$ctrl.loaded" class="services-view-container mobile-{{$ctrl.mobileView}}-view">
     <div class="add-methods">
       <h1>Browse Catalog</h1>
-      <ul class="add-other" ng-if="$ctrl.onDeployImageSelected || $ctrl.onFromFileSelected">
-        <li ng-if="$ctrl.onDeployImageSelected">
-          <a href="" ng-click="$ctrl.onDeployImageSelected()">Deploy Image</a>
-        </li>
-        <li ng-if="$ctrl.onFromFileSelected">
-          <a href="" ng-click="$ctrl.onFromFileSelected()">Import <span class="import-file-types">YAML / JSON</span></a>
-        </li>
-      </ul>
+      <div ng-if="$ctrl.onDeployImageSelected || $ctrl.onFromFileSelected">
```

Comment:
I think in practice you will either have all 3 (running in the console) or none. @jeff-phillips-18 OK to leave as is?